soda_machine.rb

```diff
@@ -7,12 +7,19 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.length
   end
 
   def find_soda(soda_brand)
+    if @sodas.include?
+    @sodas[@sodas.index(soda_brand).to_i]
```

Comment:
What kind of thing does `@sodas` have in it?  What kind of thing is `soda_brand`?  What does `.index` do?